dspot-web/README.md

```diff
@@ -0,0 +1,45 @@
+# Introduction
+
+This is a prototype web interface of Dspot, inspired from [CommitGuru](http://commit.guru/).
+
+Author: Henry Luong
+
+## Screenshots
+
+TODO add screenshots and explanation.
+
+## Running
+
+```
+cd dspot-web
+npm install --no-optional
+npm run-script start
+```
+
+## Screenshots
+
+### Front page
+![Pic 1](https://github.com/Tailp/dspot/blob/web/dspot-web/screenshots/pic1.png)
+![Pic 2](https://github.com/Tailp/dspot/blob/web/dspot-web/screenshots/pic2.png)
+### All amplified repos page
+![Pic 3](https://github.com/Tailp/dspot/blob/web/dspot-web/screenshots/pic3.png)
+### Demo submission
+
+A repo has been submitted
+
+![Pic 4](https://github.com/Tailp/dspot/blob/web/dspot-web/screenshots/pic4.png)
```

Comment:
pic 4: the "ack screen" should not contain the form and the list of recent repositories

Reply:
Hi , I have some troubles understanding this. Could it be that you mean if I submit a repo then both the form and a list dissappear ?